src/serialization/graph_binary_serialization.cpp

```diff
@@ -108,7 +120,7 @@ void parseEdge(const AttributeFactory<EdgeAttributes>& factory,
 
 void writeLayer(const SceneGraphLayer& graph, std::vector<uint8_t>& buffer) {
   BinarySerializer serializer(&buffer);
-  serializer.write(graph.id);
+  serializer.write(graph.id.layer);  // we don't save partition IDs
```

Comment:
Is there a reason not to save these? Just thinking of confusion potential if partitions get re-shuffeled/indexed during I/O.

Reply:
This serialization is distinct from the graph serialization. Layers can't be constructed standalone with a partition ID. I guess this would be a problem if you wanted to serialized a bunch of partitions from a graph independent of saving the graph (I think I punted on this because I didn't want to branch on file version)